services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -279,11 +277,10 @@ export async function getServiceImageTag(
  * @param res  - The response object
  * @param service  - The name of the service to deploy
  * @param tag  - The service image tag to deploy
- * @returns a Promise containing the deployment status
  */
 export async function execDeployScript(
```

Comment:
Is there any reason for this function to be `async`? The`exec` call uses a callback, so you won't be waiting for it. I think you can remove the `async` keyword and the return type.

Also, I don't think `res` is used here, unless I'm missing it.

Reply:
Good point! I am going to remove the async and res parameter.